.github/workflows/ci.yml

```diff
@@ -16,3 +16,7 @@ jobs:
         restore-keys: bundle
     - run: bundle install --jobs 4 --retry 3 --deployment
     - run: bundle exec rake
+    - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v1.0.7
+        with:
+          file: ./coverage/coverage.xml
```

Comment:
```suggestion
      uses: codecov/codecov-action@v1.0.7
      with:
        file: ./coverage/coverage.xml
```

This was erroring: https://github.com/alphagov/govuk-coronavirus-vulnerable-people-form/actions/runs/123839661

Reply:
😱 have updated!